kube-burner/hcp-density-aks/hcp-density-aks.yml

```diff
@@ -0,0 +1,60 @@
+---
+global:
+  indexerConfig:
+    esServers: [http://elastic-elk.apps.rsevilla-ocp45-ovn.perf-testing.devcluster.openshift.com]
+    insecureSkipVerify: false
+    defaultIndex: kube-burner
+    type: elastic
+  measurements:
+    - name: podLatency
+  clusterHealth: true
+
+jobs:
+  - name: hcp-density-aks-single-deployment
+    jobIterations: 1
+    qps: 20
+    burst: 20
+    namespacedIterations: true
+    namespace: hcp-density-aks
+    podWait: false
+    waitWhenFinished: true
+    objects:
+      - objectTemplate: templates/deployment.yaml
+        replicas: 25
+        inputVars:
+          name: deployment-1pod
+
+      - objectTemplate: templates/deployment.yaml
+        replicas: 2
+        inputVars:
+          name: deployment-2pods
+
+      - objectTemplate: templates/deployment.yaml
+        replicas: 45
+        inputVars:
+          name: deployment-3pods
+
+      - objectTemplate: templates/deployment.yaml
+        replicas: 6
+        inputVars:
+          name: deployment-pod
```

Comment:
can we also add a few multi-container pod ?

Reply:
done, ptal